### PR TITLE
[REVIEW] SEO improvements

### LINKED ISSUE #151: Width LS hero logo on LS productpage on mobile
it doesn't fit on mobile

Tested Chrome + safari 

http://www.defacto.nl/learningspaces
http://www.defacto.nl/de/learningspaces

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] H1's on every page
- [x] custom meta descriptions helper (thanks @sn3p)
- [x] make use of custom meta descriptions
- [x] fix #151 
- [x] some nicer URL's for a few pages
- [x] remove duplicate content from e-learning page
- [x] make the CMT page available through internal linking (has now 20 inlinks, so that's nicer)
- [x]  better custom meta descriptions
- [x] research the impact of redirects on sitemap.xml and so forth
- [x] remove unneeded links from sitemap builder 
- [x] correct use of links on e-learning pages

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -133,6 +137,15 @@ def page_title(page, appendCompanyName=true)
     return "Defacto - Developing People"
   end
 
+  # Use frontmatter for meta description
+  def meta_description(page=current_page)
```
Comment (fatboypunk) [inline config.rb]: `page=current_page` -> `page = current_page`

Comment (RonaldDijkstra) [inline config.rb]: Thanks

File: config.rb [hunk 2]
```diff
@@ -133,6 +137,15 @@ def page_title(page, appendCompanyName=true)
     return "Defacto - Developing People"
   end
 
+  # Use frontmatter for meta description
+  def meta_description(page = current_page)
+    return page.data.description.send(I18n.locale) if
+      page.data.description.is_a?(Hash) && page.data.description[I18n.locale]
+    return page.data.description if page.data.description
+    return Nokogiri::HTML(page.summary(250)).text if is_blog_article?
+    return t("head.default_description")
```
Comment (fatboypunk) [inline config.rb]: last `return` doens't need to be here

Comment (RonaldDijkstra) [inline config.rb]: Thanks for checking

**Comment by RonaldDijkstra** [comment]
This PR is ready for review.

File: config.rb [hunk 3]
```diff
@@ -143,7 +143,6 @@ def meta_description(page = current_page)
       page.data.description.is_a?(Hash) && page.data.description[I18n.locale]
     return page.data.description if page.data.description
     return Nokogiri::HTML(page.summary(250)).text if is_blog_article?
-    return t("head.default_description")
```
Comment (fatboypunk) [inline config.rb]: The `t("head.default_description")` can stay though, just the return could be removed ^^

Comment (RonaldDijkstra) [inline config.rb]: :bow:

File: source/convenant-medische-technologie.erb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: In 3 stappen geen zorgen meer rondom Medische Technologie
+description: Hoe gaat uw organisatie de kwaliteitsregistratie organiseren? Met het Kwaliteitspaspoort van Defacto
```
Comment (fatboypunk) [inline source/convenant-medische-technologie.erb]: this looks like a sentence that's not completed

File: source/localizable/360-feedback.html.erb [hunk 5]
```diff
@@ -1,5 +1,8 @@
 ---
 title: 360 Feedback
+description:
+  nl: Moeiteloos evaluaties uitvoeren in een gebruiksvriendelijke cloudservice. Kijk in de spiegel met 360° Feedback.
+  de: Das Tool für eine effiziente Personalentwicklung. 360° Feedback schafft Transparenz zwischen dem Vorgesetzten und seinen Mitarbeitern
```
Comment (fatboypunk) [inline source/localizable/360-feedback.html.erb]: we don't need english for this?

File: source/localizable/e-learning.html.erb [hunk 6]
```diff
@@ -1,5 +1,8 @@
 ---
 title: E-Learning
+description:
+  nl: Met Defacto als partner wordt de toepassing van e-learning in uw organisatie onherroepelijk een succes
```
Comment (fatboypunk) [inline source/localizable/e-learning.html.erb]: `een onherroepelijk succes`? also I'm not sure how good the `onherroepelijk` will do

File: source/localizable/360-feedback.html.erb (hunk 5 above)
Comment (dkhgh) [inline source/localizable/360-feedback.html.erb]: The english website is currently still the old one.

Comment (fatboypunk) [inline source/localizable/360-feedback.html.erb]: My bad

File: source/localizable/e-learning.html.erb (hunk 6 above)
Comment (RonaldDijkstra) [inline source/localizable/e-learning.html.erb]: This is based on the text on this overview page: http://www.defacto.nl/diensten/ 

Could be changed though

File: source/convenant-medische-technologie.erb (hunk 4 above)
Comment (RonaldDijkstra) [inline source/convenant-medische-technologie.erb]: Yes, it does. Let's make it better.

File: source/localizable/e-learning.html.erb (hunk 6 above)
Comment (RonaldDijkstra) [inline source/localizable/e-learning.html.erb]: We could remove `onherroepelijk`

Comment (fatboypunk) [inline source/localizable/e-learning.html.erb]: Yeah it's not my call, but it looks kinda weird and expensive ^^

Comment (RonaldDijkstra) [inline source/localizable/e-learning.html.erb]: Agreed :+1:

**Comment by RonaldDijkstra** [comment]
Set the status of the PR back to [WIP]. I will come up with better meta descriptions and review the effect on sitemap.xml of new URL's.

**Comment by RonaldDijkstra** [comment]
Reviewable again. Updated the meta's, some titles, checked the sitemap.

File: source/localizable/elearning/_elearning.de.md.erb [hunk 7]
```diff
@@ -11,7 +11,7 @@ Natürlich können wir das Entwickeln von E-Learning anstelle einer Co-Kreation
 > E-Learning-Content von Defacto ist SCORM-konform und daher frei zu nutzen in fast allen Lernumgebungen
 
 ### E-Learning Starterkit
-Mit dem **[E-learning Starterkit](/elearning-starterkit.html)** entwickeln Sie schnell und einfach Ihre eigenen E-Learning Module über spezifische Fachkenntnis, den Verhaltenskodex oder digitale Tests und Quizzes. Das Starterkit ist ein Komplettpaket mit Hardware, Software und Beratung.
+Mit dem **[E-learning Starterkit](/de/e-learning-starterkit)** entwickeln Sie schnell und einfach Ihre eigenen E-Learning Module über spezifische Fachkenntnis, den Verhaltenskodex oder digitale Tests und Quizzes. Das Starterkit ist ein Komplettpaket mit Hardware, Software und Beratung.
```
Comment (fatboypunk) [inline source/localizable/elearning/_elearning.de.md.erb]: does this work now?

File: source/sitemap.xml.builder [hunk 8]
```diff
@@ -9,7 +9,11 @@ xml.urlset "xmlns" => "http://www.sitemaps.org/schemas/sitemap/0.9" do
     page.metadata[:options][:lang] == I18n.locale &&
     !(page.data.robots && page.data.robots.include?("noindex")) &&
     !(page.path =~ /CNAME.html/) &&
-    !(page.path =~ /workshop-convenant-mt\.html/)
+    !(page.path =~ /workshop-convenant-mt\.html/) &&
+    !(page.path =~ /capp\.html/) &&
+    !(page.path =~ /elearning\.html/) &&
+    !(page.path =~ /elearning-starterkit\.html/) &&
+    !(page.path =~ /hosting\.html/)
```
Comment (fatboypunk) [inline source/sitemap.xml.builder]: Do we have to do this even when we're redirecting?

File: source/localizable/elearning/_elearning.nl.md.erb [hunk 9]
```diff
@@ -11,7 +11,7 @@ Natuurlijk kunnen wij het ontwikkelen van E-Learning ook op ons nemen. Onze onde
 > E-Learningcontent van Defacto is SCORM-compliant en dus vrij te gebruiken in bijna alle leeromgevingen.
 
 ### E-Learning Starterkit
-Met de **[E-learning Starterkit](elearning-starterkit.html)** ontwikkel je betaalbaar, snel en eenvoudig E-learning modules. Maak E-Learning gericht op specifieke vakkennis, gedragscodes of zelfs digitale toetsen. De Starterkit bestaat uit een totaalpakket van hardware, software en consultancy.
+Met de **[E-learning Starterkit](e-learning-starterkit.html)** ontwikkel je betaalbaar, snel en eenvoudig E-learning modules. Maak E-Learning gericht op specifieke vakkennis, gedragscodes of zelfs digitale toetsen. De Starterkit bestaat uit een totaalpakket van hardware, software en consultancy.
```
Comment (fatboypunk) [inline source/localizable/elearning/_elearning.nl.md.erb]: I see a `/` for every url except here, why? ^^

**Comment by fatboypunk** [comment]
Maybe it's better in the future to make even smaller PR's they can get merged faster which will make you happy and it's easier to do reviewing

File: source/sitemap.xml.builder (hunk 8 above)
Comment (RonaldDijkstra) [inline source/sitemap.xml.builder]: No we don't, thnx

**Comment by fatboypunk** [comment]
Merged since it helped at a lot of places, should be reviewed tomorrow as well and try all changes

**Pull request merged**